templates/index.html

```diff
@@ -178,7 +178,8 @@ <h4>Defender</h4>
         //  0(Attacker Graph)
         //  1(Defender Graph)
         //  2(Current Step in Graph)
-        //  3(Current Score)
+        //  3(Attacker Score)
+        //  4(Defender Score)
```

Comment:
@Chrisnguyen3 , can you update the UI to show defender score when looking at the defender view? Array data should already be updated (I think).